evoHike.Backend/Controllers/TrailsController.cs

```diff
@@ -0,0 +1,21 @@
+﻿using evoHike.Backend.Data;
+using evoHike.Backend.Models;
+using evoHike.Backend.Services;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
```

Comment:
`using Microsoft.EntityFrameworkCore; ` miért van szükség? Nem láttom, hogy bárhol is használva lenne